pkg/apis/eventing/v1alpha1/subscription_types.go

```diff
@@ -160,40 +163,18 @@ type ResultStrategy struct {
 	Target *corev1.ObjectReference `json:"target,omitempty"`
 }
 
-type SubscriptionConditionType string
-
-const (
-	// SubscriptionReady is when the From,Call and Result have been reconciled successfully.
-	SubscriptionReady SubscriptionConditionType = "Ready"
-)
-
-// SubscriptionCondition describes the state of a subscription at a point in time.
-type SubscriptionCondition struct {
-	// Type of subscription condition.
-	Type SubscriptionConditionType `json:"type"`
-	// Status of the condition, one of True, False, Unknown.
-	Status v1.ConditionStatus `json:"status"`
-	// The reason for the condition's last transition.
-	Reason string `json:"reason,omitempty"`
-	// A human readable message indicating details about the transition.
-	Message string `json:"message,omitempty"`
-}
+var subCondSet = duck.NewLivingConditionSet()
 
 // SubscriptionStatus (computed) for a subscription
 type SubscriptionStatus struct {
 	// Represents the latest available observations of a subscription's current state.
 	// +patchMergeKey=type
 	// +patchStrategy=merge
-	Conditions []SubscriptionCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	Conditions duck.Conditions `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
```

Comment:
Does the `patchStrategy` tag work for this type?

Reply:
It looks like it does! Because Conditions is a slice